src/main/java/maksym/fedorenko/bookstore/dto/UpdateBookRequestDto.java

```diff
@@ -0,0 +1,49 @@
+package maksym.fedorenko.bookstore.dto;
+
+import jakarta.validation.constraints.Positive;
+import java.math.BigDecimal;
+import lombok.Data;
+import maksym.fedorenko.bookstore.model.Book;
+import org.hibernate.validator.constraints.Length;
+
+@Data
+public class UpdateBookRequestDto {
+    @Length(max = 128)
+    private String title;
+
+    @Length(max = 128)
+    private String author;
+
+    @Length(max = 128)
+    private String isbn;
+
+    @Positive
+    private BigDecimal price;
+
+    @Length(max = 1024)
+    private String description;
+
+    @Length(max = 128)
+    private String coverImage;
+
+    public void updateBook(Book book) {
```

Comment:
dto should not contain such methods

DTO - Data Transfer Object, in other words a simple container for some data, which means it should not be aware of `Book` and the mapping process